Example/Tests/OMHSerializerTests.m

```diff
@@ -1148,7 +1148,7 @@
                          @"header.schema_id.name": @"body-fat-percentage",
                          @"header.schema_id.namespace":@"omh",
                          @"header.schema_id.version": @"1.0",
-                         @"body.body_fat_percentage.value": @23.2,
```

Comment:
Error message for .232 / 23.2 is "expected (__NSCFNumber): 23.2, got (__NSCFNumber): 23.2".

The assertion line that fails:
`expect([object valueForKeyPath:keyPath]).to.equal(keyPathValue);`

I'm thinking this must be some edge case but in the ExpectA cocoapod lib being used.